src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -25,6 +25,7 @@ const getButtonDataSeAttr = function(authenticator) {
 };
 
 /* eslint complexity: [0, 0] */
+// eslint-disable-next-line max-statements
```

Comment:
is it needed to disable this lint rule?

Reply:
This is not needed as i moved some of the statements inside switch hence removed it.